### DISCUSSION TITLE
[Feature Request] Add ImGui Menu Scale option

### DISCUSSION THREAD

**Discussion by SuperBro64** [HarbourMasters/2ship2harkinian#613] [Ideas]
On SoH, the F1 menu bar was too small on my laptop display when at Normal size. Increasing it to Large size made the text much easier to read.

Meanwhile, on 2Ship, the default, unchangeable F1 menu bar currently displays on my laptop at the same size as Normal on SoH, making it a little too small for me. Adding a ImGui Menu Scale option to 2Ship like on SoH, or something similar, would be very much appreciated.

Just to showcase my issue visually, this is a screen snip of my laptop display in its entirety with 2Ship running:
![image](https://github.com/HarbourMasters/2ship2harkinian/assets/102717364/84400dd8-434c-43e7-aad6-25da6b6a6156)

And here is my whole laptop display with SoH running and the F1 menu bar size set to Large:
![image](https://github.com/HarbourMasters/2ship2harkinian/assets/102717364/e399a287-270c-4be6-817e-e953e38a6968)

As can be seen, it is (hopefully) apparent how much smaller the default size is on 2Ship on my display, compared to larger sizes on SoH.

EDIT: Changing my laptop's display resolution from its default of 2560 x 1600 to 1920 x 1200 increased the menu bar size and it's much better for me now. Still, an scaling option would be nice just in general.

**Comment by dCo3lh0** [comment]
lmao, just had the same issue but on arch linux, the HUD from IMGUI is absurdly small in my 4K monitor.

**Comment by Channy-736** [comment]
I also would appreciate this and I think many others would too. Playing Majora’s Mask on the big screen tv, the UI is so tiny I have to stand and walk up to the tv to see it.

**Comment by DarkHeros09** [comment]
yes please add this feature, I'm having the same problem.

**Comment by cwjbowler** [comment]
My poor old eyesight needs this too. Playing on a 4K TV, with the distance away from the screen I normally sit, the menu is unreadable.

**Comment by T1g** [comment]
i am squinting so hard someone save me from this hell

**Comment by Eblo** [comment]
Implemented with #1709
